config/locales/en.yml

```
@@ -445,6 +445,7 @@ en:
           format: "%{message}"
           virus_found: "'%{attribute}' contains a virus - choose a different file"
           scan_error: "An error has occurred, try again in a few minutes"
+          too_big: "The selected file must be smaller than 1Mb"
```

```suggestion
          too_big: "The selected file must be smaller than 1MB"
```

i _think_ `Mb` means megabit which is teeny tiny 😆